documentation/smg/02_development_environment.md

```diff
@@ -1 +1,102 @@
+  * Click Connect to establish the connection. This may need refreshing daily, or after shutdown or sleep, or just sometimes. Check back to the VPN page; if the Connect button is visible, you are not connected even if your email is working.
+
+#### Getting MATLAB
+
+MATLAB can be found at STFC RAL IT Support (Knowledge Base/MATLAB) at www.facilities.rl.ac.uk/support/KnowledgeBase/Matlab.aspx. This should set up an account with Matlab with your STFC email as id and a password of your choice. You can then go to the Matlab download page for STFC and download; choose R2018b or R2019b, which are the current versions supported at the time of writing. Or update to more recent if we have changed. This will give you an installation .exe.
+
+Running the .exe will download the files and install. You may be asked for activation details from Matlab. This will be your id and password as above. You are getting an academic individual STFC licence. Activation may also be required on first launch. Ideally you will be logged in as you with appropriate privileges. If not, you may need to run as administrator; not doing so puts you in an infinite loop of activate, launch, activate, launch.....
+
+This appears to be a one-user license and I am not necessarily expecting that you will be able to install on multiple machines e.g. personal and work machine. But it may work.
+
+NB I had problems trying to restart the download - it said I did not have access rights. Clearing cookies from the page solved this.
+
+#### Using MATLAB
+
+##### Prerequisites
+
+* You need to install `cmake`. I installed the current (at time of writing) version 3.18.2; there is a minimum version but all current versions should be OK for this. Select that `cmake` is on the path.
+* You need to install the Visual Studio SDK. This can be the actual SDK or a working VS installation. I installed Visual Studio 16 2019 community edition (it's free). As (currently) cloned from github, Horace will cache a reference to VS 2017 and will say that it cannot find VS. In explorer or otherwise, go to Horace/build in the Horace repo and delete or rename CMakeCache.txt before building. 
+* You need to build Horace and Herbert. With both cloned as in the instructions above, open a Powershell and cd to the main Horace repo directory. Then type
+  * `.\tools\build_config\build.ps1 -build -vs_version 2019`
+* The build will tailor the `horace_on` and `herbert_on` commands to fit the location of the current repo.
+* You can now access Horace: cd to Horace\build\local_init and type `horace_on`. To avoid this indirect access, you can set this directory to be searched by setting the environment variable MATLABPATH to the full directory path before launching MATLAB. `horace_on` can then be used immediately on launch.
```

Comment:
May be worth mentioning you should `cd` into the directory from within the Matlab command window; we were in PowerShell in the previous bullet points

Reply:
Good point.